### PR TITLE
Add GitHub buttons + navigation to submodel templates

### PR DESCRIPTION
**Pull request by monsieuremre**
Changes:

* Added the option navigate between two documentation websites directly within the UI. It is meant to be the main way of navigating between aas-specs and submodels documentation sites. As of now, the submodels site is a deadlink, but it will work as soon as the submodel website gets up.

* Added new button to directly report an issue from the website
* Added new button to directly see the history of the current source from the website

* Added logo for the history button
* Added logo for the issue report button
* Added logo for the edit this page button
* Added logo for the generate pdf button

* Created uniform theming for all the buttons

* Small fixes and changes

### CODE REVIEW

**Comment by monsieuremre** [comment]
Lightbox modal view is not included in the request as it is not  yet ready.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
   activate:
     runs-on: ubuntu-latest
     if: |
-      github.repository == 'admin-shell-io/aas-specs-antora-ui' && !startsWith(github.event.head_commit.message, 'Release ')
+      github.repository == 'rwth-iat/aas-specs-antora-ui' && !startsWith(github.event.head_commit.message, 'Release ')
```
Comment (zrgt) [inline .github/workflows/release.yml]:
```suggestion
      github.repository == 'admin-shell-io/aas-specs-antora-ui' && !startsWith(github.event.head_commit.message, 'Release ')
```

File: package.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "aas-specs-antora-ui",
   "description": "An archetype project that produces a UI for creating documentation sites with Antora",
-  "homepage": "https://github.com/admin-shell-io/aas-specs-antora-ui/blob/main/package.json",
+  "homepage": "https://github.com/rwth-iat/aas-specs-antora-ui/blob/main/package.json",
```
Comment (zrgt) [inline package.json]:
```suggestion
  "homepage": "https://github.com/admin-shell-io/aas-specs-antora-ui/blob/main/package.json",
```

File: package.json [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
   "license": "MPL-2.0",
   "repository": {
     "type": "git",
-    "url": "https://github.com/admin-shell-io/aas-specs-antora-ui/blob/main/package.json"
+    "url": "https://github.com/rwth-iat/aas-specs-antora-ui/blob/main/package.json"
```
Comment (zrgt) [inline package.json]:
```suggestion
    "url": "https://github.com/admin-shell-io/aas-specs-antora-ui/blob/main/package.json"
```

**Review by zrgt — CHANGES_REQUESTED** [review]

**Review by zrgt — APPROVED** [review]

**Pull request merged**